client/metrics.go

```diff
@@ -0,0 +1,144 @@
+package client
+
+import (
+	"context"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/ory/fosite"
+	"github.com/ory/x/metricsx"
+)
+
+type Metrics struct {
+	prometheus.Collector
+	metricsx.Observer
+	Manager
+
+	Clients              prometheus.Gauge
+	ClientsCreated       prometheus.Counter
+	ClientsAuthenticated metricsx.CounterVec
+}
+
+func (m *Metrics) GetConcreteClient(ctx context.Context, id string) (*Client, error) {
+	return m.Manager.GetConcreteClient(ctx, id)
```

Comment:
You don't need to re-implement this. Outer structs directly inherit methods from their embedded structs

Reply:
Done! :)